lib/internal/child_process.js

```diff
@@ -610,9 +610,6 @@ function setupChannel(target, channel) {
         this.emit('error', ex);  // FIXME(bnoordhuis) Defer to next tick.
       }
     }
-
-    /* If the master is > 2 read() calls behind, please stop sending. */
-    return channel.writeQueueSize < (65536 * 2);
```

Comment:
It's currently an unused value returned from `_send()` and swallowed by `send()` and has been since 0.12.0. The last time end users were able to get a boolean returned from `send()` was 0.10.40. I'm not arguing that you're wrong. I'm just providing additional info/context in case that changes your estimation of the code's importance. (I'm guessing it does not and I certainly value your judgment on it more than my own. You have internalized a ton of historical context I am oblivious to.)